cppapi/server/multiattribute.cpp

```diff
@@ -1045,7 +1048,10 @@ void MultiAttribute::remove_attribute(string &attr_name,bool update_idx)
 	if (update_idx == true)
 	{
 		for (;pos != attr_list.end();++pos)
+		{
 			(*pos)->set_attr_idx((*pos)->get_attr_idx() - 1);
+			ext->attr_map[(*pos)->get_name_lower()].att_index_in_vector--;
```

Comment:
It is better to split this line declaring `string attr_name_lower` variable - better for debugging and maintaining